src/app/products/manage-tax-components/create-tax-component/create-tax-component.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="taxComponentForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Name</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="taxComponentForm.controls.name.hasError('required')">
+              Name is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Percentage</mat-label>
+            <input type="number" matInput required formControlName="percentage">
+            <mat-error *ngIf="taxComponentForm.controls.percentage.hasError('required')">
+              Percentage is <strong>required</strong>
+            </mat-error>
+            <mat-error *ngIf="taxComponentForm.controls.percentage.hasError('min')">
+              Percentage <strong>cannot</strong> be smaller than zero
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Credit Account Type</mat-label>
+            <mat-select formControlName="creditAccountType" (selectionChange)="populateCreditAccount()">
+              <mat-option *ngFor="let creditAccountType of creditAccountTypeData" [value]="creditAccountType.id">
+                {{ creditAccountType.value }}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Credit Account</mat-label>
+            <mat-select formControlName="creditAcountId">
+              <mat-option *ngFor="let creditAccount of creditAccountData" [value]="creditAccount.id">
+                {{ creditAccount.name }}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+
+        </div>
+
+        <mat-form-field>
+          <mat-label>Start date</mat-label>
+          <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="startDatePicker" required formControlName="startDate">
+          <mat-datepicker-toggle matSuffix [for]="startDatePicker"></mat-datepicker-toggle>
+          <mat-datepicker #startDatePicker></mat-datepicker>
+          <mat-error *ngIf="taxComponentForm.controls.startDate.hasError('required')">
+            Start date is <strong>required</strong>
+          </mat-error>
+        </mat-form-field>
```

Comment:
This field should also cover the entire length. The `div` is incorrectly placed. Please check.